src/frontend/app/features/service-catalog/service-summary/service-summary.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="summary">
+  <app-tile-grid fit="true">
+    <app-tile-group>
+      <app-tile>
+        <app-service-summary-card></app-service-summary-card>
```

Comment:
In order to not repeat the service name (header title and card title), we could replace the title here with `Summary`, which would match the behaviour in cf/org/space summary tabs